content/07.pathogenesis.md

```diff
@@ -10,12 +10,20 @@ This section would also be great for the introduction of zoonotic diseases which
+
+Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions. In China, R(t) was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10.1016/S1473-3099(20)30144-4]. R(t) decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after. Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%. An independent study came to similar conclusions: in a two-week period before Jan 23 finding R(t)=2.38, and decreasing to R(t) = 1.34 (using data from Jan 24 to Feb 3) or R(t)=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221]. In South Korea, R(t) was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf]. Metro data was also analyzed to estimate the effects of social distancing measures. R(t) decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced. In Seoul, R(t) decreased slightly, but remained close to 1 (and larger than R(t) in Daegu). This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R(t) remains above 1, thus secondary outbreaks are possible. It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do. 
```

Comment:
```suggestion
Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions. 
In China, R(t) was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10.1016/S1473-3099(20)30144-4]. 
R(t) decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after. 
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%. 
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R(t)=2.38, and decreasing to R(t) = 1.34 (using data from Jan 24 to Feb 3) or R(t)=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221]. 
In South Korea, R(t) was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf]. 
Metro data was also analyzed to estimate the effects of social distancing measures. 
R(t) decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced. 
In Seoul, R(t) decreased slightly, but remained close to 1 (and larger than R(t) in Daegu). 
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R(t) remains above 1, thus secondary outbreaks are possible. 
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do. 
```